vm-migration/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1.0.57"
+hypervisor = { path = "../hypervisor" }
```

Comment:
I'm concerned about the dependency here. `vm-migration` is a very low level crate and I would expect the `hypervisor` crate to rely on it rather than the other way around.

Reply:
The latest code have already don't need this dependency.